shared/menubar/index.render.desktop.js

```diff
@@ -88,19 +94,19 @@ class Render extends Component<void, Props, State> {
         username={this.props.username} />]
     }
 
-    const styles = this.state.showingPublic ? stylesPublic : stylesPrivate
+    const styles = this.state.showingPrivate ? stylesPrivate : stylesPublic
 
     const mergedProps = {
       ...this.props,
       smallMode: true,
       private: newPrivate,
       public: newPublic,
-      onSwitchTab: showingPublic => this.setState({showingPublic})
+      onSwitchTab: showingPublic => this.setState({showingPrivate: !showingPublic})
```

Comment:
Er, nevermind! The broken syntax highlighting made it hard to spot the `!` here.